src/DynamoCore/DynamoCore.csproj

```diff
@@ -380,43 +379,54 @@ limitations under the License.
   <ItemGroup>
     <Resource Include="BuiltInAndOperators\BuiltIn.Migrations.xml" />
   </ItemGroup>
-  <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Choose>
+    <When Condition=" '$(OS)' != 'Unix' ">
```

Comment:
What is this section for?

Reply:
Is to decide which powershell to use, on the Linux image for the mono-build validation we have Powershell Core available, otherwise we use the windows powershell